frontend/dist/assets/tracim/appInterface.js

```diff
@@ -1,9 +1,16 @@
-(function () {
-  let prevSelectedAppFeature = ''
-  let prevSelectedAppFullScreen = ''
-
-  getSelectedApp = name => {
-    switch (name) {
+var prevSelectedAppFeature = ''
+var prevSelectedAppFullScreen = ''
+var APP_NOT_LOADED = 'appNotLoaded'
+var TIME_TO_RETRY = 500
+var RETRY_TIMEOUT = 5000
```

Comment:
what is the usage of TIME_TO_RETRY and RETRY_TIMEOUT?

Reply:
TIME_TO_RETRY is time to wait until the next try to open the app

Reply:
Could you put `60000` as `RETRY_TIMEOUT`